it/utils/ViewHelpers.scala

```diff
@@ -28,6 +31,19 @@ trait ViewHelpers { self: AnyWordSpec with Matchers =>
   val ENGLISH = "English"
   val WELSH = "Welsh"
 
+  val ExpectedResults: Object
+  val Selectors: Object
+
+  def urlGet(url: String, welsh: Boolean = false, follow: Boolean = true, headers: Seq[(String, String)] = Seq())(implicit wsClient: WSClient): WSResponse = {
+
```

Comment:
Minor point but will help remove duplication, you can if/else on the header value, so its either "en" or "cy" then we wont need two wsclient calls.